sourcecode/apis/common/app/Auth/AppTokenAuthenticator.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Auth;
+
+use App\Models\AccessToken;
+use App\Models\Application;
+use Illuminate\Http\Request;
+use Psr\Log\LoggerInterface;
+
+class AppTokenAuthenticator
+{
+    private LoggerInterface $logger;
+
+    public function __construct(
+        LoggerInterface $logger
```

Comment:
Can be a promoted property (`__construct(private LoggerInterface $logger)`) to avoid a bit of boilerplate.